Index: cctools-port-dbcdc0d36eeb47a7ac4b7d5abe772886fec52e56/cctools/include/foreign/ppc/types.h
===================================================================
--- cctools-port-dbcdc0d36eeb47a7ac4b7d5abe772886fec52e56.orig/cctools/include/foreign/ppc/types.h
+++ cctools-port-dbcdc0d36eeb47a7ac4b7d5abe772886fec52e56/cctools/include/foreign/ppc/types.h
@@ -90,9 +90,9 @@ typedef	int			int32_t;
 typedef	unsigned int		u_int32_t;
 #ifndef _INT64_T
 #define _INT64_T
-typedef	long long		int64_t;
+typedef	long int64_t;
 #endif
-typedef	unsigned long long	u_int64_t;
+typedef	unsigned long u_int64_t;
 
 #if __LP64__
 typedef int64_t			register_t;
